### ISSUE TITLE
Methodically address all the Bullet issues

### ISSUE THREAD

**Issue by asideofcode-dev** [codebar/planner#1885]
## Description of the issue 📄

As part of the pull request to update to Rails 5, https://github.com/codebar/planner/pull/1882, Bullet has started throwing exceptions on certain queries. For the moment we have set `Bullet.raise = false` in `config/environments/test.rb`. Bullet is still turned on it just logs any offences instead of raising exceptions.

## Steps to fix 🛠

Methodically go through all queries raising Bullet exceptions, and address the issue. It's almost always something to do with n+1 queries or eager loading.

## To do 📋

* [ ] Claim this issue (comment below, or assign yourself if you are part of the codebar org)
* [ ] Fork and clone the repository
* [ ] Update the relevant files. Follow the steps to fix section in this issue.
* [ ] Commit your changes as one commit. Use the title of this issue as your commit message
* [ ] Submit a pull request
* [ ] Mention this issue in the PR description by including it's number
* [ ] Have your pull request reviewed & merged by a codebar team member

**Comment by gnclmorais** [comment]
Definitely, a long time ago I looked into a few of these queries, while I was trying to update Rails version. Have you heard of [`query_diet`](https://github.com/makandra/query_diet#query-diet-) as well? I [read](https://twitter.com/ilrock__/status/1668530435909885952) some interesting things about it…
